progress/window.c

```diff
@@ -162,19 +162,19 @@ static int progress_window_repaint(struct MuttWindow *win)
 
   if (wdata->size == 0)
   {
-    message_bar(wdata->win, wdata->display_percent, "%s %zu (%d%%)", wdata->msg,
+    message_bar(wdata->win, wdata->display_percent, _("%s %zu (%d%%)"), wdata->msg,
```

Comment:
Please can you add a `L10N` comment, before each line, explaining what each of the fields is.
(same for the two below)